docs/Configuration.md

```diff
@@ -2267,3 +2267,21 @@ This option allows comments in `package.json`. Include the comment text as the v
   }
 }
 ```
+
+### `workerIdleMemoryLimit` \[number]
```

Comment:
Might be better to move the whole option description after `watchman`, but before `//`.